server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -136,155 +129,77 @@ def generate_summary_histogram():
     Return:
         Dash html div element containing request time to close histogram.
     """
-    histogram = html.Div(
-        # Histogram for the request timeToClose.
+    return html.Div(
         dcc.Graph(
             id="time_close_histogram", style={"height": "40vh",
                      "width": "48vw"}
         ), style=merge_dict(CHART_OUTLINE_STYLE, {"width": "48vw",
                  "height": "40vh"}))
-    return histogram
 
 
-def generate_council_name_dropdown():
+def generate_council_name_dropdown(output_id):
     """Generates the neighborhood council (nc) dropdown for the 
     comparison dashboard.
 
     This functions generates the html elements for the 
-    nc dropdown for the left side of the comparison dashboard.
+    nc dropdown for the comparison dashboard.
 
```

Comment:
Please add `Args` section since we added an arg now.

Reply:
Thanks for the catch - resolved